### PR TITLE
Release/3.19.2

### PR DESCRIPTION
**Pull request by MouazAlzahabi**
Handles #3107

### CODE REVIEW

File: Multisig/Cross-layer/FirebaseRemoteConfig.swift [hunk 1]
```diff
@@ -17,13 +17,16 @@ class FirebaseRemoteConfig {
         case deprecated
         case safeClaimEnabled
         case crashDebugEnabled
+        //FIXME Remove feature flag and flag handling after release
+        case intercomMigration
```
Comment (elgatovital) [inline Multisig/Cross-layer/FirebaseRemoteConfig.swift:21]: we have released, should we remove the flag now?

File: Multisig/Cross-layer/FirebaseRemoteConfig.swift [hunk 2]
```diff
@@ -17,13 +17,16 @@ class FirebaseRemoteConfig {
...
     }
 
     private var remoteConfig: RemoteConfig
     private let defaultValues: [String : NSObject] = [Key.newestVersion.rawValue : "" as NSObject,
                                                       Key.deprecatedSoon.rawValue : "" as NSObject,
                                                       Key.deprecated.rawValue : "" as NSObject,
-                                                      Key.safeClaimEnabled.rawValue : false as NSObject]
+                                                      Key.safeClaimEnabled.rawValue : false as NSObject,
+                                                      Key.intercomMigration.rawValue : false as NSObject] //FIXME Remove feature flag and flag handling after release
```
Comment (elgatovital) [inline Multisig/Cross-layer/FirebaseRemoteConfig.swift:29]: the migration is done, we don't need it anymore, right?

File: Multisig/Cross-layer/FirebaseRemoteConfig.swift (hunk 1 above)
Comment (MouazAlzahabi) [reply Multisig/Cross-layer/FirebaseRemoteConfig.swift:21]: No, next pr

File: Multisig/Cross-layer/FirebaseRemoteConfig.swift (hunk 2 above)
Comment (MouazAlzahabi) [reply Multisig/Cross-layer/FirebaseRemoteConfig.swift:29]: No, next pr

File: Multisig/Cross-layer/FirebaseRemoteConfig.swift (hunk 1 above)
Comment (elgatovital) [reply Multisig/Cross-layer/FirebaseRemoteConfig.swift:21]: is there a task for this?

Comment (MouazAlzahabi) [reply Multisig/Cross-layer/FirebaseRemoteConfig.swift:21]: No, I wanted to do it as part of increasing the version, if you want make a ticket for it

Comment (elgatovital) [reply Multisig/Cross-layer/FirebaseRemoteConfig.swift:21]: yes, please make a ticket. We also need to remove this value from firebase remote config.

Comment (MouazAlzahabi) [reply Multisig/Cross-layer/FirebaseRemoteConfig.swift:21]: Will make it later

**Review by elgatovital — APPROVED** [review]

**Pull request merged**